datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -461,20 +461,25 @@ def tag(check, version, push, dry_run):
         checks = [check]
 
     for check in checks:
+        echo_success('Check `{}`'.format(check))
+
         # get the current version
         if not version:
             version = get_version_string(check)
 
         # get the tag name
         release_tag = get_release_tag_string(check, version)
-        echo_info('Tagging HEAD with {}'.format(release_tag))
+        echo_waiting('Tagging HEAD with {}'.format(release_tag))
 
         if dry_run:
             version = None
             continue
 
         result = git_tag(release_tag, push)
-        if result.code != 0:
+
+        if result.code == 128 or 'already exists' in result.stderr:
+            echo_warning('Tag `{}` already exists, skipping...'.format(release_tag))
```

Comment:
I actually think it may make sense to fail here. If the tag already exists, then it could easily mean we have a tag pointing to the wrong commit that needs to be addressed. Do you have an example where it you hit this that was safe to ignore?

Reply:
When tagging all checks at once

Reply:
So my worry here is more geared towards if we move this to some automation. If some automation job succeeds, noone is going to check the logs. Like I mentioned, if we hit a real issue where a tag was inappropriately set to the wrong commit, then we should fail in that case. 

However, causing it to fail would make the task no longer idempotent because we may be successful for checks A, B, and C, then fail on D. So you fix D, rerun the job, and now we fail on A because the tag was already set. 

I personally think the ideal solution here would be to find a way to keep track of the checks that need releasing, OR make this a continuable task, so if we have to fix D, we can run the task with a continue flag that only ran D and later. That being said, I don't think we will have to do this often/for much longer (run the release task in batch) so that may not be worth the effort. 
Can we leave a note in the code to mention maybe not automate this until something like this is in place ^.? I think its likely fine to merge as-is with some note since this won't be a frequently run task.